src/middlewared/middlewared/plugins/consulkv.py

```diff
@@ -112,6 +138,31 @@ class ConsulService(Service):
 
         return await self.reload()
 
+    def _aws_config_file(self, region=None, key_id=None, access_key=None):
+        config_path = '/root/.aws/config'
+        credentials_path = '/root/.aws/credentials'
+        aws_vault = '/root/.aws/'
```

Comment:
There is no chance, mostly because AWS SDK looking for the .aws directory in the user home that spawn the process that will try to authenticate with AWS services.